ckanext/canada/schemas/dataset.yaml

```diff
@@ -471,10 +471,10 @@ resource_fields:
 
 - field_name: schema
   label: Schema
-  preset: hidden_in_form
+  preset: resource_schema
 - field_name: validation_options
   label: Validation options
-  preset: hidden_in_form
```

Comment:
yeah, but won't work without it either. The `validation_options` are empty and validation can't process without it. We can re-visit it so that it works for both portal and registry